tensorflow/tools/tf_windows_build_dockerfiles/README.md

```diff
@@ -0,0 +1,23 @@
+**To build a Windows container locally, execute these commands. These commands have been tested on Windows PowerShell when run as an administrator:**
+```
+md C:\tf
+md C:\tf\tmp
+cd C:\tf
+git clone https://github.com/Bobarshad/tensorflow.git
```

Comment:
This should be tensorflow/tensorflow, not your personal repository.

Reply:
Done